networking_arista/ml2/arista_sec_gp.py

```diff
@@ -371,8 +371,12 @@ def _create_acl_rule(self, in_cmds, out_cmds, sgr, security_group_ips=None):
         Deals with multiple configurations - such as multiple switches
         """
         # Only deal with valid protocols - skip the rest
-        if not sgr or sgr['protocol'] not in SUPPORTED_SG_PROTOCOLS or (
-            sgr['ethertype'] not in SUPPORTED_SG_EHTERTYPES):
+        if not sgr or sgr['protocol'] not in SUPPORTED_SG_PROTOCOLS:
+            return in_cmds, out_cmds
+
+        if sgr['ethertype'] is None:
+            sgr['ethertype'] = SUPPORTED_SG_ETHERTYPES[0]
```

Comment:
I fear, you are modifying the model here.
That gets then persisted to the database.